apps/calendar/src/controller/column.ts

```diff
@@ -122,13 +177,63 @@ function setRenderInfo(
     comingEnd,
     startColumnTime,
     endColumnTime,
+    duplicateEvents: uiModel.duplicateEvents,
   };
+}
+
+function setRenderInfo(
+  uiModel: EventUIModel,
+  columnIndex: number,
+  baseWidth: number,
+  startColumnTime: TZDate,
+  endColumnTime: TZDate
+) {
+  const renderInfoOptions = getRenderInfoOptions(
+    uiModel,
+    columnIndex,
+    baseWidth,
+    startColumnTime,
+    endColumnTime
+  );
 
   setDimension(uiModel, renderInfoOptions);
   setInnerHeights(uiModel, renderInfoOptions);
   setCroppedEdges(uiModel, renderInfoOptions);
 }
 
+function setDuplicateEvents(uiModels: EventUIModel[], options: typeof collapseDuplicateEvents) {
+  const { getDuplicateEvents, sortDuplicateEvents, getMainEvent } = options;
+
+  const eventObjects = uiModels.map((uiModel) => uiModel.model.toEventObject());
+
+  uiModels.forEach((targetUIModel) => {
+    if (targetUIModel.collapse || targetUIModel.duplicateEvents.length) {
```

Comment:
```suggestion
    if (targetUIModel.collapse || targetUIModel.duplicateEvents.length > 0) {
```

여기도 명시적으로 `> 0` 해주는거 어떠신가요?